MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -271,27 +271,27 @@ private void FetchDataFromTags(Audio audio)
                     audio.Genres = tags.Genres.Distinct(StringComparer.OrdinalIgnoreCase).ToArray();
                 }
 
-                if (!audio.ProviderIds.TryGetValue("MusicBrainzArtist", out var artistValue) || artistValue == null)
+                if (!audio.TryGetProviderId("MusicBrainzArtist", out var artistValue) || string.IsNullOrEmpty(artistValue))
```

Comment:
You do not need to check for null nor emptiness. If TryGetProviderId returns true, then it is a valid provider id.